sigs.yaml

```
@@ -3044,7 +3044,8 @@ committees:
   name: Code of Conduct
   mission_statement: >
     The Kubernetes Code of Conduct Committee (CoCC) is the body that is responsible
-    for enforcing and maintaining the Kubernetes Code of Conduct.
+    for enforcing and maintaining the Kubernetes Code of Conduct. Please submit private
+    incident reports to the conduct@kubernetes.io mailing list.
```

```suggestion
    for enforcing and maintaining the Kubernetes Code of Conduct.
```
As this is the general mission statement for the group, it may be better to leave the contact portion out of it.

Just an FYI - you will also need to run `make` or if you don't have go installed, but have access to docker run `make generate-dockerized` which should take care of regenerating the various docs. 👍